src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private double base1;
+    private double base2;
+    private double height;
+    private double side;
+
+    public IsoscelesTrapezoid(String color, double base1, double base2, double side) {
+        super(color);
+        this.base1 = base1;
+        this.base2 = base2;
+        if (base1 < base2) {
+            double temp = base1;
+            this.base1 = this.base2;
+            this.base2 = temp;
+        }
```

Comment:
according from your var names, there's no difference between side `base1` and `base2`. better remove this logic, or rename them like `bottom` and `top`
```suggestion
        if (base1 < base2) {
            double temp = base1;
            this.base1 = this.base2;
            this.base2 = temp;
        }
```